src/main/java/websocket/models/File.java

```diff
@@ -0,0 +1,71 @@
+package websocket.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class File {
+	@JsonProperty("FileID")
+	public long fileID;
+	
+	@JsonProperty("Name")
+	public String fileName;
+	
+	@JsonProperty("RelativePath")
+	public String relativePath;
+	
+	@JsonProperty("Version")
+	public long fileVersion;
+	
+	@JsonProperty("ProjectID")
+	public long projectID;
+
+	public File(long fileID, String fileName, String relativePath, long fileVersion, long projectID) {
+		super();
+		this.fileID = fileID;
+		this.fileName = fileName;
+		this.relativePath = relativePath;
+		this.fileVersion = fileVersion;
+		this.projectID = projectID;
+	}
```

Comment:
not sure if it matters, but the server also keeps track of a `CreationDate` on the files